articles/managed-operations/man-ref-customer-prereqs.md

```diff
@@ -0,0 +1,132 @@
+
+UKCloud will provide you with the VPN standards, including the pre-shared key, peer endpoint and peer subnet to configure the VPN.
+
+## Anti-Virus as a Service
+
+Anti-Virus as a Service provides you with the confidence that your environments are protected from the dangers of viruses and malware, all without the usual headaches associated with the deployment, management, and operation of an antivirus solution. For more information see the [*Anti-Virus as a Service Service Scope*](man-sco-antivirus.md).
+
+UKCloud will provide you with the installation *package(s)* and commands to install and configure the antivirus agent on your virtual machines.
+
+## Patching as a Service
+
+Patching as a Service provides routine OS patch management for compute instances. It helps you accelerate business value by removing the burden of basic OS hygiene, letting you focus your attention above the operating system. For more information, see the [*Patching as a Service Service Scope*](man-sco-patching.md).
+
+### Operating systems
+
+UKCloud only supports in-life, vendor-backed operating systems, such as Microsoft Windows, Red Hat Enterprise Linux and Canonical Ubuntu. Customer-built operating systems (for example Gentoo Linux custom build) are unsupported by this service.
+
+### Customer environment configuration
+
+#### Linux
+
+To provide secure communication between UKCloud's centralised patch management platform and your virtual machine, we use key-based SSH sessions, reducing the need to share credentials. As such, you'll need to fulfil the following prerequisites on each virtual machine subscribing to this service:
+
+- An allow rule for port 22 (SSH) on your virtual machine's firewall (for example, iptables).
+
+- A local user account with passwordless sudo access. You'll need to share details of this account with UKCloud when onboarding to this service.
+
+- The `/etc/ssh/sshd_config` configuration file must be configured to allow SSH access for the patching user you created in the step above.
+
+- UKCloud will provide you with a public SSH key during the onboarding process. You'll need to append this key to the `authorized_keys` file, located at `~/.ssh/authorized_keys` within the home directory of the user you created in the step above.
+
+#### Windows
+
+For UKCloud to correctly scan for and apply operating system patches, you must agree to enable and configure the Server Message Block (SMB) protocol by ensuring ports 445 and 139 (both over TCP) are open on all virtual machines and associated operating systems opted-in to this service. Furthermore, there must be a bi-directional rule allowing port 3121 (over TCP) on your edge appliance and operating system firewall.
+
+Furthermore, several other prerequisites are required. 
+
+- You must create a patching user, which must:
+
+  - Be an admin on the virtual machine
+
+  - Have access to the Control Panel
+
+  - Filer and Print Sharing enabled for all networks
```

Comment:
Is this associated with the patching user? In which case make sure the wording of the bullet item matches the others and flows from the lead-in sentence:

- Have Filer and Print Sharing enabled for all networks

If it's not associated with the patching user, it needs to be a level up and change to:

- You must enable Filer and Print Sharing for all networks

Happy to make the change if you just tell me which one it needs to be...

Reply:
I believe this is the second one, so shouldn't be specific to the user.

Reply:
Fixed in commit c35444a9adc277fd9817a0dca4fde84b7c9776fe.